scribus/text/storytext.cpp

```diff
@@ -198,21 +199,22 @@ void StoryText::moveCursorLeft(bool isGrapheme)
 {
 	if (isGrapheme)
 	{
-		int pos;
-		QTextBoundaryFinder boundaryFinder(QTextBoundaryFinder::Grapheme, plainText());
+		BreakIterator* it = getGraphemeIterator();
+		if (!it)
+			return;
+
+		int pos = cursorPosition();
+		it->setText(plainText().utf16());
```

Comment:
Since we always call `setText()` after getting the iterator, I’d move to the `get*Iterator()` method and pass `painText()` to it. This also makes sure no use site misses setting the text.